autoinstrumentation/nodejs/src/autoinstrumentation.ts

```diff
@@ -35,6 +36,7 @@ function getMetricReader() {
 const sdk = new NodeSDK({
     autoDetectResources: true,
     instrumentations: [getNodeAutoInstrumentations()],
+    textMapPropagator: new AWSXRayPropagator(),
```

Comment:
Agreed.  The javascript SDK must recognize [all of these values](https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_propagators) to meet spec.  @pichlermarc do know know how the `OTEL_PROPAGATORS` env var interacts with `NodeSDK`?  If `NodeSDK` doesn't honor `OTEL_PROPAGATORS` that feels like a bug.

Reply:
`NodeSDK` does honor `OTEL_PROPAGATORS` for the internal values (`tracecontext, baggage, b3, b3multi, jaeger`), via the [NodeTracerProvider](https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-sdk-trace-node/src/NodeTracerProvider.ts#L37) that is used in the SDK.. The third party options `xray` and `ottrace` are available in the [js-contrib](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/propagators/opentelemetry-propagator-aws-xray) repo and currently only added via code, not yet available as an environment variable option.

Reply:
@JamieDanielson thanks for the details. @ademar-prothon at this point I don't think there is anything we should change in the operator. Instead it would be good to add this functionality to opentelemetry-js directly so that I can enable the xray propagator via the env variable.

Once that feature is release from opentelemetry-js we can update the operator'a dependencies.